### PR TITLE
Improve docs related to actor specs

### PR DESCRIPTION
**Pull request by mhamas**
This PR is adding improvements related to actor specs initiative.

Namely it
1) fixes usage of `apify.json` to` .actor/actor.json`
2) adds info on where the input schema is read from
3) adds info on where the docker is read from
4) some minor tweaks in formatting

### CODE REVIEW

**Review by TheoVasilis — APPROVED** [review]

File: content/docs/actors/development/input_schema.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ paths:
 
 An actor's input schema defines the input that the actor accepts and the UI components used for input in Apify Console. Using input schema, you can provide and easy-to-use UI for your actor's users and also ensure that the input provided is valid.
 
-Input schema must be stored in a file named **INPUT_SCHEMA.json** in the actor's root directory. The file's maximum size is 100 kB. If the input schema is provided, then input is always validated to fulfill the schema when an actor is being started (via API or from the console).
+Input schema can be embedded as an object in the `.actor/actor.json` file under `input` field. Alternatively, you can store a path to a json file in that field. If the `input` field or `.actor/actor.json` is omitted, input schema from `.actor/INPUT_SCHEMA.json` is used. In the absence of that file, the `INPUT_SCHEMA.json` file stored in the actor's root directory is used. The file's maximum size is 100 kB. If the input schema is provided, then input is always validated to fulfill the schema when an actor is being started (via API or from the console).
```
Comment (m-murasovs) [inline content/docs/actors/development/input_schema.md:18]:
```suggestion
Input schema can be embedded as an object in the **.actor/actor.json** file under the `input` field. Alternatively, you can store a path to a JSON file in that field. If the `input` field or **.actor/actor.json** are omitted, the input schema from **.actor/INPUT_SCHEMA.json** is used. In the absence of that file, the **INPUT_SCHEMA.json** file stored in the actor's root directory is used. The file's maximum size is 100 kB. If the input schema is provided, then input is always validated to fulfill the schema when an actor is being started (via API or from Apify Console).
```

File: content/docs/actors/development/output_schema.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ An actor's output schema defines the structure and both API and visual represent
 
 ## How to organize files in the .actor folder: two options
 
-**A)** all config options are being set in a .actor/actor.json file, e.g.:
+**A)** all config options are being set in a `.actor/actor.json` file, e.g.:
```
Comment (m-murasovs) [inline content/docs/actors/development/output_schema.md:21]:
```suggestion
**A)** all config options are being set in a **.actor/actor.json** file, e.g.:
```

File: content/docs/actors/development/output_schema.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ An actor's output schema defines the structure and both API and visual represent
 }
 ```
 
-**B)** .actor/actor.json links to other sub-config files in the same folder, e.g.:
+**B)** `.actor/actor.json` links to other sub-config files in the same folder, e.g.:
```
Comment (m-murasovs) [inline content/docs/actors/development/output_schema.md:46]:
```suggestion
**B)** **.actor/actor.json** links to other sub-config files in the same folder, e.g.:
```

File: content/docs/actors/development/source_code.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ Similarly, as with the [Git repository]({{@link actors/development/source_code.m
 
 ## [](#custom-dockerfile)Custom Dockerfile
 
-Internally, Apify uses Docker to build and run actors. To control the build of the actor, you can create a custom **Dockerfile** in the root of the Git repository or Zip directory. If the **Dockerfile** is missing, the system uses the following default:
+Internally, Apify uses Docker to build and run actors. To control the build of the actor, you can create a custom **Dockerfile** and either reference from `dockerfile` field in actor's config at `.actor/actor.json`, or store it at `.actor/Dockerfile` or at `Dockerfile` in the actor's root directory. These three sites are searched for in this order of preference. If the **Dockerfile** is missing, the system uses the following default:
```
Comment (m-murasovs) [inline content/docs/actors/development/source_code.md:64]:
```suggestion
Internally, Apify uses Docker to build and run actors. To control the build of the actor, you can create a custom **Dockerfile** and either reference from the `dockerfile` field in the actor's config in **.actor/actor.json**, or store it in **.actor/Dockerfile** or **Dockerfile** in the actor's root directory. These three sites are searched for in this order of preference. If the **Dockerfile** is missing, the system uses the following default:
```

**Review by m-murasovs — CHANGES_REQUESTED** [review]
Just some formatting suggestions - keeping the "`" for code segments and bold for file names, etc.

**Review by m-murasovs — APPROVED** [review]

**Pull request merged**